app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, file_copy = command.split(" ")
+
+    if file == file_copy or cp != "cp":
```

Comment:
`It must do nothing in case the user is trying to copy file to file with the same name.`